lmms_eval/tasks/gmai/gmai.yaml

```
@@ -0,0 +1,31 @@
+dataset_path: louis-mart/GMAI-MMBench-val
+
+task: "gmai"
+dataset_kwargs:
+  token: True
+test_split: validation
+output_type: generate_until
+doc_to_visual: !function utils.gmai_doc_to_visual
+doc_to_text: !function utils.gmai_doc_to_text
+doc_to_target: "answer"
+process_results: !function utils.gmai_process_results
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)